### ISSUE TITLE
Installation of 0.13.0 on Ubuntu 22.10 failing

### ISSUE THREAD

**Issue by tschiex** [BradyAJohnston/MolecularNodes#112]
**Describe the bug**
The installation of atomium seems to succeed but is reported to fail. Then MolecularNodes won't accept to do anything :-)

**To Reproduce**
1. Install blender.org 3.4 using the tar.xz ball
2. Start blender and install MolecularNodes addon (version 0.13)
3. Ask to install Atomium
4. A strange dialog pops-up on the shell console, saying that atomium installation failed (but it didn't).

**Expected behavior**
Atomium should be reported as successfully installed?

**Error Codes**

```
Read prefs: /home/tom/.config/blender/3.4/config/userpref.blend
Looking in links: /tmp/tmphje50gu2
Requirement already satisfied: setuptools in /usr/lib/python3/dist-packages (59.6.0)
Requirement already satisfied: pip in /home/tom/.local/lib/python3.10/site-packages (22.3.1)
Requirement already satisfied: pip in /home/tom/.local/lib/python3.10/site-packages (22.3.1)
Requirement already satisfied: atomium in ./3.4/python/lib/python3.10/site-packages (1.0.11)
Requirement already satisfied: rmsd in /home/tom/.local/lib/python3.10/site-packages (from atomium) (1.4)
Requirement already satisfied: msgpack in ./3.4/python/lib/python3.10/site-packages (from atomium) (1.0.4)
Requirement already satisfied: paramiko in /usr/lib/python3/dist-packages (from atomium) (2.10.4)
Requirement already satisfied: numpy in /usr/lib/python3/dist-packages (from atomium) (1.21.5)
Requirement already satisfied: valerius in /home/tom/.local/lib/python3.10/site-packages (from atomium) (0.2.0)
Requirement already satisfied: requests in /usr/lib/python3/dist-packages (from atomium) (2.27.1)
Requirement already satisfied: scipy in /usr/lib/python3/dist-packages (from rmsd->atomium) (1.8.1)
Error: Atomium installation failed.
```

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.10 
 - Hardware: x86 PC
 - Blender Version: 3.4
- MolecularNodes Version: 0.13

**Comment by BradyAJohnston** [comment]
Hmm, unsure what the issue might be here. Inside of Blender's pythong console are you able to import Atomium:
```python]
import atomium as at
```

Does this succeed? If it does then it the internal check is breaking somehow. 

Is there are reason you are not wanting to use the more recent 2.0 of MN? As I can't spend much dev time on previous versions

**Comment by tschiex** [comment]
With some lack of confidence, my guess would be that this comes from the mixed set of paths that are used for installation. Some packages were already installed by the Ubuntu system (in /usr/lib/python3/), some were already installed in my user directory (/home/tom/.local/lib/python3.10/site-packages) and some got installed in the blender-local python install path by your installation process (in ./3.4/python/lib/python3.10/site-packages). This is the case for atomium for example.

So "import atomium" on my native python install does not work (as expected). But if I go inside the blender folder and use the blender python interpreter, it imports smoothly:

blender-3.4.0-linux-x64 > ./3.4/python/bin/python3.10 
Python 3.10.8 (main, Oct 24 2022, 20:47:11) [GCC 9.3.1 20200408 (Red Hat 9.3.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import atomium
>>> 

Don't spend time on this if you don't see an easy patch. I'm sorry to not be able to wiggle my side-chains :-) but I will cope with the frustration and use 2.0.

**Comment by BradyAJohnston** [comment]
The wiggle atoms is next on the list to be added to 2.1, so should be available next week! If I can I want to avoid working on 0.13 and with atomium because it ended up being a problematic mess (which is why I switched away from it and re-wrote the whole package). Glad 2.0 is working well though!

**Comment by tschiex** [comment]
I have played a bit more with my setup: in Blender, in the scripting console, the sys.path of the python interpreter does not include the global system path (/usr/lib/python3//usr/lib/dist-packages) while it is clearly discovered and used during the atomium installation procedure of MolecularNodes 0.13 (as the log above shows). This explains the issue.
 
Once I did a sys.path.append("/usr/lib/python3/dist-packages") in the blender console, clicking on the "install atomium" button in the addon worked and the addon seems to work too. Ready to wiggle :-)
